src/node/gov/gov_endpoint_registry.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ccf/common_endpoint_registry.h"
+#include "node/gov/api_version.h"
+#include "node/gov/handlers/acks.h"
+#include "node/gov/handlers/proposals.h"
+#include "node/gov/handlers/recovery.h"
+#include "node/gov/handlers/service_state.h"
+#include "node/gov/handlers/transactions.h"
+#include "node/share_manager.h"
+
+namespace ccf
+{
+  // Eventually, this should extend BaseEndpointRegistry, rather than
+  // CommonEndpointRegistry. But for now, we still support the old gov API by
+  // extending this, and that includes the common endpoints
+  class GovEndpointRegistry : public CommonEndpointRegistry
```

Comment:
Worth highlighting this implementation pattern.

We can't have the old `MemberEndpointRegistry` and the new `GovEndpointRegistry` as separately installed frontends/registries, because they both want to own the `/gov` prefix. So what I've currently got is `Member extends Gov extends Common extends Base`, so that `/gov` exposes:
- `/gov/ack` (from Member, for people still using the old API)
- `/gov/members/state-digests/{memberId}:ack` (from Gov, part of new API)
- `/gov/commit` (from Common, for people still using the old API)

When the old API is no longer supported, we remove `Member` and instantiate `Gov` directly. At that point I think we can and should remove the `Common` intermediate class here, since the common APIs aren't part of the documented `/gov` API and I don't think they're necessary: The endpoints which _are_ necessary for basic governance interaction, such as `/commit` and `/tx`, have replacements in the new API, and the other endpoints can be accessed on `/app` or `/node`.